en/micro-integrator/docs/references/connectors/smpp-connector/smpp-inbound-endpoint-example.md

```diff
@@ -0,0 +1,131 @@
+# SMPP Inbound Endpoint Example
+
+The SMPP inbound endpoint allows you to consume messages from SMSC via WSO2 EI. WSO2 EI SMPP inbound endpoint acts as a message consumer. It creates a connection with the SMSC, then listen a port to consume only SMS messages from the SMSC and injects the messages to the EI sequence. It will receive alert notification or will notify when a data short message accepted.
```

Comment:
```suggestion
The SMPP inbound endpoint allows you to consume messages from SMSC via WSO2 EI. WSO2 EI SMPP inbound endpoint acts as a message consumer. It creates a connection with the SMSC, then listens on a port to consume only SMS messages from the SMSC and injects the messages to the EI sequence. It will receive alert notifications or will notify when a data short message accepted.
```

Reply:
Updated.